### PR TITLE
Changelog for 1.1.0

### PR DESCRIPTION
**Pull request by ikoevska**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+	
+### Updated
+
+* [Updated #358](https://github.com/NativeScript/nativescript-cli/issues/358): Multiple improvements and bug fixes for iOS debugging.
+
+### Fixed
+
+* [Fixed #446](https://github.com/NativeScript/nativescript-cli/issues/446): Autocompletion might continue to work after disabling becauce `tns autocomplete disable` does not update `.profile` files.
+* [Fixed #391](https://github.com/NativeScript/nativescript-cli/issues/391): The default 9-patch image for Android projects is malformed and causes error messages in the build log.
+* [Fixed #324](https://github.com/NativeScript/nativescript-cli/issues/324): When you run commands for the `ios` platform, the NativeScript CLI might showUnable to find adb in PATH. Default one from %s resources will be used. the following unrelated adb warning: `Unable to find adb in PATH. Default one from %s resources will be used.`
```
Comment (rosen-vladimirov) [inline CHANGELOG.md]: might show unable
showUnable is concatenated at the moment

Comment (rosen-vladimirov) [inline CHANGELOG.md]: also the message looks strange

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+* [Implemented #510](https://github.com/NativeScript/nativescript-cli/issues/510): You can work with custom NativeScript plugins in your project. You can add a plugin from an npm package, a local folder, a URL or a `.tar.gz` file. For more information, run `$ tns help plugin`
+* [Implemented #362](https://github.com/NativeScript/nativescript-cli/issues/362): You can use npm as package manager for your NativeScript projects.
+	
+### Updated
+
+* [Updated #358](https://github.com/NativeScript/nativescript-cli/issues/358): Multiple improvements and bug fixes for iOS debugging.
+
+### Fixed
+
+* [Fixed #446](https://github.com/NativeScript/nativescript-cli/issues/446): Autocompletion might continue to work after disabling becauce `tns autocomplete disable` does not update `.profile` files.
```
Comment (teobugslayer) [inline CHANGELOG.md]: becauce -> because ;)

File: CHANGELOG.md (hunk 1 above)
Comment (teobugslayer) [inline CHANGELOG.md]: "showUnable to find adb in PATH. Default one from %s resources will be used. the" looks like a typo

**Comment by teobugslayer** [comment]
:+1:

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by Fatme** [comment]
:+1:

**Pull request merged**